parser.c

```diff
@@ -142,49 +158,52 @@ static int parse_readbuf_string(struct tinyiiod *iiod, char *str)
 		return -EINVAL;
 
 	tinyiiod_do_readbuf(iiod, device, (size_t) bytes_count);
+
 	return 0;
 }
 
-int tinyiiod_parse_string(struct tinyiiod *iiod, char *str)
+/***************************************************************************//**
+ * @brief tinyiiod_parse_string
+*******************************************************************************/
+int32_t tinyiiod_parse_string(struct tinyiiod *iiod, char *str)
 {
-	while (*str == '\n' || *str == '\r')
-		str++;
-
 	if (str[0] == '\0')
 		return 0;
 
-	if (!strncmp(str, "VERSION", sizeof("VERSION"))) {
+	if (!strncmp(str, "VERSION", sizeof("VERSION") - 1)) {
```

Comment:
this looks like a fix commit;
typically, fix commits should be separate, and have an explanation of what they fix